src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationFormFooter.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  Button,
+  PaneFooter,
+} from '@folio/stripes/components';
+
+const BatchGroupConfigurationFormFooter = ({
```

Comment:
does it differ so much from a common acq FormFooter?

Reply:
Common `FormFooter` includes `Save` and `Cancel` buttons only